client/client.go

```diff
@@ -0,0 +1,105 @@
+package client
+
+import (
+	"errors"
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type Client struct {
+	conn *grpc.ClientConn
+}
+
+func New(address string) *Client {
+	conn, err := grpc.Dial(address)
+	if err != nil {
+		log.Fatalf("Cannot connect: %v", err)
+	}
+	return &Client{conn: conn}
+}
+
+func (c *Client) Write(
+	ctx context.Context, name string, offset int64, data []byte, isAppend bool,
+) (int64, error) {
+	pbClient := pb.NewCfsClient(c.conn)
+	reply, err := pbClient.Write(
+		ctx,
+		&pb.WriteRequest{Name: name, Offset: offset, Data: data, Append: isAppend},
+	)
+
+	if err != nil {
+		return 0, err
+	}
+	return reply.BytesWritten, pbErr2GoErr(reply.Error)
+}
+
+func (c *Client) Read(
```

Comment:
one line. there is no 80 line requirement in go. if you use far more than 80 chars, then there is something wrong with your approach. :)